### PR TITLE
Adding unit test for SqlGetHandler query building change

### PR DESCRIPTION
**Pull request by masonchester**
Adding unit test for get request query building parameters for version 0.3.17.

### CODE REVIEW

**Comment by masonchester** [comment]
This test is very ugly FYI prepare your eyes.

File: src/test/java/io/rcktapp/api/handler/sql/SqlGetHandlerTest.java [hunk 1]
```diff
@@ -75,14 +75,20 @@ public class SqlGetHandlerTest {
     @Captor
     ArgumentCaptor<List> queryParametersCaptor;
 
+    @Captor
+    ArgumentCaptor<String> sqlCaptor;
+    
+    @Captor
+    ArgumentCaptor<Map<String, String>> rqlParamsCaptor;
+
```
Comment (paulboeck) [inline src/test/java/io/rcktapp/api/handler/sql/SqlGetHandlerTest.java]: Looks like you added these variables, but they're not used? Or am I missing something?

Comment (eriknj) [reply src/test/java/io/rcktapp/api/handler/sql/SqlGetHandlerTest.java]: or, if they are, perhaps you've forgotten to delete in-line assignments to those references?

File: src/test/java/io/rcktapp/api/handler/sql/SqlGetHandlerTest.java [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+            rqlMockedStatic.when(() -> Rql.getRql(sqlDb.getType())).thenReturn(sqlRql);
+
+            when(api.getCollection(request.getCollectionKey(), SqlDb.class)).thenReturn(collection);
+            sqlMockedStatic.when(() -> Sql.getInClauseStr(J.explode(",", Sql.check(request.getEntityKey())))).thenReturn(passedQuery);
+
+            // When we call the service method on the mocked handler actually call the real method
+            doCallRealMethod().when(handler).service(service, api, endpoint, action, chain, request, response);
+
+            // More mocked methods that return objects reused locally
+            when(sqlRql.asCol("plugins")).thenReturn(collectionName);
```
Comment (john-bartz-rp) [inline src/test/java/io/rcktapp/api/handler/sql/SqlGetHandlerTest.java]: String collectionName = "plugins";

**Review by john-bartz-rp — APPROVED** [review]

**Pull request merged**